test/transpose.js

```diff
@@ -0,0 +1,18 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+describe('transpose', function() {
+  it('returns an array of two arrays', function() {
+    var input = [['a', 1], ['b', 2], ['c', 3]];
+    eq(R.transpose(input), [['a', 'b', 'c'], [1, 2, 3]]);
+  });
+  it('skips elements when rows are shorter', function() {
+    var actual = R.transpose([[10, 11], [20], [], [30, 31, 32]]);
+    var expected = [[10, 20, 30], [11, 31], [32]];
+    eq(actual, expected);
+  });
```

Comment:
That's the line I was considering.  But it doesn't really seem to work.  I think perhaps the only law we could support is something to the effect of 

```
// if each list in lists has the same length, then
transpose(transpose(lists))  ≍  lists
```

And if the lists are of different lengths, you're on your own.